Makefile

```diff
@@ -0,0 +1,22 @@
+GOPATH=$(shell go env GOPATH)
+GOBIN=$(GOPATH)/bin
+GOSRC=$(GOPATH)/src
+
+$(GOBIN)/golangci-lint:
+	curl -sfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh| sh -s -- -b $(GOBIN) v1.36.0
```

Comment:
Even thought this is the "official" and documented way of installing `golangci-lint`, we should avoid curling and piping bash scripts in general: from a security perspective, it leaves us vulnerable to supply-chain attacks like [codecov](https://about.codecov.io/security-update/).

Easy fixes: 
- either perform integrity check on the downloaded script before executing it
- or use `go get github.com/golangci/golangci-lint/cmd/golangci-lint@<COMMIT HASH>`

Reply:
Good catch @svalentino 
I followed the same pattern (AKA copy/paste) other security tools were doing but indeed this bash-curl is a flaw path.
Fixed. 👍

Reply:
Awesome 👍